install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -286,13 +279,18 @@ data:
   cluster-id: "{{ .Values.global.cluster.id }}"
 {{- end }}
 
-{{- if .Values.global.gke.enabled }}
-
-  ipam: "kubernetes"
+  # Encapsulation mode for communication between nodes
+  # Possible values:
+  #   - disabled
+  #   - vxlan (default)
+  #   - geneve
+{{- if .Values.gke.enabled }}
```

Comment:
this should be `Values.global.gke.enabled`

Reply:
That's not what I see in the PR that was backported - https://github.com/cilium/cilium/pull/14153/files